src/dash-table/derived/table/fragmentStyles.ts

```diff
@@ -16,10 +16,12 @@ export default (
     viewport: IDerivedData,
     rowPadding: IViewportPadding,
     scrollbarWidth: number
-): { fragment?: CSSProperties, cell?: CSSProperties}[][] => {
+): { fragment?: CSSProperties, cell?: CSSProperties }[][] => {
+    const base = [{}, { fragment: { marginRight: scrollbarWidth } }];
+
     if (!virtualization || !uiCell || !uiViewport) {
         return [
-            [{}, {}],
+            base,
```

Comment:
If there is no virtualization or if the virtualization is not ready -- return the scrolling offset only